QKSMS/src/main/java/com/moez/QKSMS/common/QksmsRateSnack.java

```diff
@@ -0,0 +1,14 @@
+package com.moez.QKSMS.common;
+
+import org.ligi.snackengage.conditions.AfterNumberOfOpportunities;
+import org.ligi.snackengage.conditions.NeverAgainWhenClickedOnce;
+import org.ligi.snackengage.snacks.RateSnack;
+
+/* https://github.com/ligi/SnackEngage
+*  "This shows a snackbar after app was started *20* times and never again when once clicked on Rate" */
+
+public class QksmsRateSnack extends RateSnack {
```

Comment:
This class should also override `getText()` and `getActionText` so we control the strings, and can support localization later on

For `getText` I'd make it return a string saying "Like QKSMS?"

And for `getActionText`, "Rate"

Of course, make them string resources :)